### PR TITLE
feat(logs): remove play logs button in console logs

### PR DESCRIPTION
**Pull request by ctjhoa**
# What does this PR do?

https://qovery.atlassian.net/browse/FRT-1260

### CODE REVIEW

**Comment by RemiBonnet** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery []() preview b1cc6d03-1b9e-4536-b5ee-dc186c55218c | [dev](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/b1cc6d03-1b9e-4536-b5ee-dc186c55218c/services/general) |
 /qovery []() preview cc1de7de-94e5-40f8-8e45-bc8986d9dfec | [storybook](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/cc1de7de-94e5-40f8-8e45-bc8986d9dfec/services/general) |
 /qovery []() preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery []() preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> Don’t Sacrifice Readability, you are going to debug it in 6 months

File: libs/shared/console-shared/src/lib/layout-logs/layout-logs.tsx [hunk 1]
```diff
@@ -259,6 +268,19 @@ export function LayoutLogs({
...
+      {pauseLogs && newMessagesAvailable && (
+        <Button
+          className="absolute bottom-5 left-1/2 flex w-72 -translate-x-1/2 items-center justify-center text-sm"
+          variant="solid"
+          radius="full"
+          size="md"
+          type="button"
+          onClick={() => setPauseLogs?.(false)}
+        >
+          New logs&nbsp;
```
Comment (RemiBonnet) [inline libs/shared/console-shared/src/lib/layout-logs/layout-logs.tsx:280]: Remove this character and using margin

**Review by RemiBonnet — CHANGES_REQUESTED** [review]

File: libs/shared/console-shared/src/lib/layout-logs/layout-logs.tsx (hunk 1 above)
Comment (ctjhoa) [reply libs/shared/console-shared/src/lib/layout-logs/layout-logs.tsx]: Fixed https://github.com/Qovery/console/pull/1587/commits/11c12ebdcf1b2a432051375fdd4ad50d0154f7ff

**Review by RemiBonnet — APPROVED** [review]

**Pull request merged**